test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test

```diff
@@ -117,10 +117,15 @@ echo
 01. PUT /v2/entities/E1/attrs/A1 and see it fail
 ================================================
 HTTP/1.1 404 Not Found
-Content-Length: 0
+Content-Length: 61
+Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
+{
```

Comment:
This change in .test (and some other similar) corresponds to the following entry in CNR: "Fix: missing error payload for NotFound error in **'PUT /v2/entities/<eId>/attrs/<attrName>**' and 'PUT /v2/entities/<eId>/attrs/<attrName>/value' operations".

NTC (just a remark).